src/translator.js

```diff
@@ -0,0 +1,97 @@
+const timeTracker = {
+  second: 0,
+  minute: 0,
+  hour: 0,
+  day: 0,
+  year: 0,
+}
+
+const translateTime = (num) => {
+  if(num === 0) {
+    return "now";
+  }
+  if (num < 0) {
+    getSeconds(num * -1);
+    return `${timeTrackerReader()} ago`
+  }
+  console.log('before', timeTracker)
+  getSeconds(num);
+  return timeTrackerReader();
+}
+
+const getSeconds = (num) => {
+  if(num/60 >= 1) {
+    getMinutes(num);
+  } else {
+    timeTracker.second = num;
+  }
+}
+
+const getMinutes = (num) => {
+  if(num/3600 >= 1) {
+    getHours(num);
+  } else {
+    timeTracker.minute = Math.floor(num/60)
+    const leftOver = num % 60
+    if(leftOver > 0) { getSeconds(leftOver) }
+  }
+}
+
+const getHours = (num) => {
+  if(num/86_400 >= 1) {
+    getDays(num);
+  } else {
+    timeTracker.hour = Math.floor(num / 3600)
+    const leftOver = num % 3600
+    if (leftOver > 0) { getMinutes(leftOver) }
+  }
+}
+
+const getDays = (num) => {
```

Comment:
I think `getMinutes`, `getHours`, and `getDays` look so similar that they could _maybe_ be structured into a single function. Maybe something like:
```js
const getTime = (num, smallDivider, largeDivider) => {
  if(num/largeDivider >= 1) {
    // this function call needs more thinking around it, though
    getTime(num);
  } else {
    timeTracker.minute = Math.floor(num/smallDivider)
    const leftOver = num % smallDivider
    if(leftOver > 0) { getSeconds(leftOver) }
  }
}
```